pkg/common/volume_operations.go

```diff
@@ -172,10 +193,24 @@ func (vo *VolumeOperationsImpl) CreateVolume(ctx context.Context, v api.Volume)
 		if err = vo.k8sClient.UpdateCRWithAttempts(ctxWithID, ac, 5); err != nil {
 			ll.Errorf("Unable to set size for AC %s to %d, error: %v", ac.Name, ac.Spec.Size, err)
 		}
+		if vo.acReservationEnabled {
```

Comment:
How will you handle race condition between Controller (CreateVolume) and Node (NodePublishVolume - inline)?

Reply:
This is old problem. We already have issue https://github.com/dell/csi-baremetal/issues/105